document/endnotes.go

```diff
@@ -0,0 +1,81 @@
+package document
+
+import (
+	"github.com/unidoc/unioffice/schema/soo/wml"
+)
+
+// Endnote is an individual footnote reference within the document.
```

Comment:
Should that be "individual endnote" ? Check also function comments below, they refer to "footnote", should be "endnote"  presumably